doc/reduce-traffic.md

```diff
@@ -48,3 +48,7 @@ Be reminded of the effects of this setting.
   wallet is loaded or if you use the node to broadcast transactions.
 - It makes block propagation slower because compact block relay can only be
   used when transaction relay is enabled.
+- This setting sets the flag "-walletbroadcast" to be "0", if it is currently
+  set to "1", which disables the automatic broadcasting of transactions
+  from inbound peers. If a peer is whitelisted, and "-whitelistforcerelay" is
```

Comment:
```suggestion
  from the wallet. If a peer is whitelisted, and "-whitelistforcerelay" is
```

Reply:
sorry if this is a super naive question...my understanding is transactions from inbound peers and wallet transactions are two different concepts. So actually both of them not relayed when -blocksonly mode is turned on?

Reply:
Yes, but walletbroadcast is a wallet specific setting and does not control incoming traffic, only outgoing (broadcast)
I think it could help to split the walletbroadcast and whitelist settings in two bullet points.